static/src/js/components/OrderStatus/OrderStatusItem.js

```diff
@@ -630,6 +630,7 @@ export class OrderStatusItem extends PureComponent {
                           retrievalId={retrievalId}
                           retrievalCollectionId={retrievalCollectionId}
                           showTextWindowActions={!isEsi}
+                          collectionIsCSDA={collectionIsCSDA}
```

Comment:
I thought I remember you looking at a test for this, that verified this new prop was being passed. That would be good to add